spec/lib/ability_spec.rb

```diff
@@ -66,7 +66,7 @@
       let(:facility) { Facility.cross_facility }
 
       it { is_expected.to be_allowed_to(:manage_accounts, facility) }
-      it { is_expected.to be_allowed_to(:manage, Account) }
+      it { is_expected.to be_allowed_to([:read, :edit], Account) }
```

Comment:
The `be_allowed_to` matcher only accepts a single ability, not an array.

We have `it_is_allowed_to([:read, :edit], Account)`, which essentially expands to
```
it { is_expected.to be_allowed_to(:read, Account) }
it { is_expected.to be_allowed_to(:edit, Account) }
```